cmd/kube-controller-manager/app/certificates.go

```diff
@@ -180,21 +180,21 @@ func startCSRApprovingController(ctx ControllerContext) (http.Handler, bool, err
 	)
 	go approver.Run(1, ctx.Stop)
 
-	return nil, true, nil
+	return approver, nil
```

Comment:
This needs a security review. Guessing the returning nil was not a lack of handler but a deliberate security precaution.

Reply:
hmm, should making all fields of `CertificateController` private enough? A reference to the controller itself does not seem to affect security in any way.

Reply:
Avoid return the pointer anyways.